Examples/BookingSystem.AspNetFramework/Stores/SessionStore.cs

```diff
@@ -76,9 +77,17 @@ protected override void GetOrderItems(List<OrderItemContext<SessionOpportunity>>
 
             // Response OrderItems must be updated into supplied orderItemContexts (including duplicates for multi-party booking)
 
+            List<OccurrenceTable> occurrenceTable;
+            List<ClassTable> classTable;
+            using (var db = FakeBookingSystem.Database.Mem.Database.Open())
```

Comment:
This is interesting... doesn't `using` close the connection? How is this still accessible afterwards?

Reply:
the references assigned are outside the using statement, this was done for simplicity, so retaining the orginal query syntax.

Reply:
Ok great!